container/rules/aggregate_debian_packages.bzl

```diff
@@ -0,0 +1,98 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
Call this file
debian_pkg_tar.bzl

Reply:
Will remove this file and extract downloading debian packages as a separate rule in docker_toolchains.bzl file